tests/suites/test_suite_nist_kw.data

```diff
@@ -0,0 +1,444 @@
+NIST KW self test
+mbedtls_nist_kw_self_test:
+
+NIST KW init #1 wrapping AES-128: OK
+depends_on:MBEDTLS_AES_C
+mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_AES:128:1:0
+
+NIST KW init #2 unwrapping AES-128: OK
+depends_on:MBEDTLS_AES_C
+mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_AES:128:1:0
+
+NIST KW init #3 CAMELLIA-256: bad cipher
+depends_on:MBEDTLS_CAMELLIA_C
+mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_CAMELLIA:256:0:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW init #4 AES-224: bad key size
+depends_on:MBEDTLS_AES_C
+mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_AES:224:1:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW init #5 BLOWFISH-128: bad cipher
+depends_on:MBEDTLS_BLOWFISH_C
+mbedtls_nist_kw_setkey:MBEDTLS_CIPHER_ID_BLOWFISH:128:0:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #1 KW plaintext OK ( 2 to 2^54 - 1 semiblocks )
+nist_kw_lengths:16:MBEDTLS_KW_MODE_KW:1:0
+
+NIST KW lengths #2 KWP plaintext OK ( 1 to 2^32 - 1 octets )
+nist_kw_lengths:5:MBEDTLS_KW_MODE_KWP:1:0
+
+NIST KW lengths #3 KW ciphertext OK ( 3 to 2^54 semiblocks )
+nist_kw_lengths:32:MBEDTLS_KW_MODE_KW:0:0
+
+NIST KW lengths #4 KWP ciphertext OK ( 2 to 2^29 semiblocks semiblocks )
+nist_kw_lengths:24:MBEDTLS_KW_MODE_KWP:0:0
+
+NIST KW lengths #5 KW plaintext too short ( 2 to 2^54 - 1 semiblocks )
+nist_kw_lengths:5:MBEDTLS_KW_MODE_KW:1:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #6 KWP plaintext too short ( 1 to 2^32 - 1 octets )
+nist_kw_lengths:0:MBEDTLS_KW_MODE_KWP:1:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #7 KWP plaintext too long ( 1 to 2^32 - 1 octets )
+nist_kw_lengths:4294967296:MBEDTLS_KW_MODE_KWP:1:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #8 KW ciphertext too short (3 to 2^54 semiblocks)
+nist_kw_lengths:16:MBEDTLS_KW_MODE_KW:0:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #9 KWP ciphertext too short ( 2 to 2^29 semiblocks semiblocks )
+nist_kw_lengths:8:MBEDTLS_KW_MODE_KWP:0:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
+
+NIST KW lengths #10 KWP ciphertext too long ( 2 to 2^29 semiblocks semiblocks )
+nist_kw_lengths:4294967297:MBEDTLS_KW_MODE_KWP:0:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
```

Comment:
This test case doesn't do what it says. On all the platforms we use, `int` is a 32-bit type so 4294967297 is 1 (assuming an environment that doesn't overflow or saturate signed integers).